docs/source/setup_lora.rst

```diff
@@ -72,7 +75,10 @@ Modifying the LoRa parameters
 
 -  After the registration was completed, the LoRa parameters can be transferred to the program.
 -  They can be set up at the configuration site of the Geiger counter (see above).
--  Go through the configuration site until the settings of the LoRa  parameters are displayed. Type in the 3 parameters from the TTN console (**APPEUI, DEVEUI, APPKEY**). They can be find in your TTN account for each device (see above). The HEX values must be entered **without** spaces as they appear in the TTN or other devices.
+-  Go through the configuration site until the settings of the LoRa  parameters are displayed. 
+   Type in the 3 parameters from the TTN console (**APPEUI, DEVEUI, APPKEY**). 
+   They can be find in your TTN account for each device (see above). 
+   The HEX values must be entered **without** spaces as they appear in the TTN or other devices.
```

Comment:
das "or other devices" passt hier irgendwie nicht, was war gemeint?

vielleicht "or other services" (TTN ist ja ein Service) oder "or other networks" (TTN ist ein Netzwerk)?

Reply:
I don't know, I just resolved the compilation warnings from @Baitcookie from #374

Reply:
@rexfue was ist mit "other devices" gemeint, stimmt das so oder ist es other ...? siehe oben.

Reply:
Diskutiert das doch bitte in einem Issue, ich hätte den Pull Request hier gerne möglichst bald durch und der Text aus dem File steht genau so seit langem auf der Seite.

Reply:
Leave 'or other devices' out - its not in the german Readme. Don't know, why  Baitcookie put it in.

Reply:
I fixed this, too.